### PR TITLE
Cps 7 admin cut - new PR

### PR DESCRIPTION
**Pull request by isentropy**
I think the branches diverged so I'm opening this new PR. AdminFee test is working

### CODE REVIEW

**Comment by isentropy** [comment]
@jtakalai I opened a new [PR](https://github.com/streamr-dev/monoplasma/pull/8) because the branches diverged. You can close this one.

File: test/truffle/BalanceVerifier.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ const { assertEqual, assertFails } = require("../utils/web3Assert")
 
 const MonoplasmaMember = require("../../src/member")
 const MonoplasmaState = require("../../src/state")
-const plasma = new MonoplasmaState(0, [], { saveBlock: () => {} })
+
+const admin = "0x0000000000000000000000000000000000000000"
```
Comment (jtakalai) [inline test/truffle/BalanceVerifier.js]: make this non-zero, because zero might be special sometimes...

Comment (jtakalai) [reply test/truffle/BalanceVerifier.js]: (see test/mocha/state.js)

File: test/truffle/Monoplasma.js [hunk 2]
```diff
@@ -4,22 +4,35 @@ const ERC20Mintable = artifacts.require("openzeppelin-solidity/contracts/token/E
...
 
 const MonoplasmaState = require("../../src/state")
-
+const MonoplasmaWatcher = require("../../src/watcher")
+const JoinPartChannel = require("../../src/joinPartChannel")
+const fileStore = require("../../src/fileStore")("/tmp/mono")
 let currentBlockNumber = 1
 
+
 contract("Monoplasma", accounts => {
```
Comment (jtakalai) [inline test/truffle/Monoplasma.js]: Maybe add ownership transfer test as well, to see it's handled correctly

**Comment by jtakalai** [comment]
Please remove or silence the logging in tests

**Pull request merged**